src/org/omegat/gui/glossary/TransTipsMarker.java

```diff
@@ -68,8 +67,9 @@
 
         List<Mark> marks = new ArrayList<Mark>();
 
+        IGlossaryRenderer entryRenderer = Core.getGlossary().getLocalRenderer();
```

Comment:
Is there really a need to fetch the renderer out of the Glossary Pane? It seems like unnecessary coupling.

Is there a reason not to simply have a separate instance here?

Reply:
OK, for me the important is to share the algorithm giving renderer by name (because else it could happen that you accidentally do a modification in GlossaryTextArea and forget to replicate it to TransTips). But if we consider that I accept the idea of creating the class GlossaryRenderers (question 1), then I think that the common method giving the selected renderer should be there.

Reply:
I've moved the preferred renderer into `GlossaryRenderers`.